includes/Core/Modules/Modules.php

```diff
@@ -882,6 +900,11 @@ private function get_module_schema() {
 					),
 					'readonly'    => true,
 				),
+				'owner'        => array(
+					'type'        => 'string',
+					'description' => __( 'Owner of the module.', 'google-site-kit' ),
+					'readonly'    => true,
+				),
```

Comment:
See above, let's make this an object with `id` and `login`.

Reply:
Yep, updated.